planpro_importer/planpro110/reader110.py

```diff
@@ -9,7 +10,28 @@ def __init__(self, plan_pro_file_name):
         if not plan_pro_file_name.endswith(".ppxml"):
             plan_pro_file_name = plan_pro_file_name + ".ppxml"
         self.plan_pro_file_name = plan_pro_file_name
+        self.root_object = parse(self.plan_pro_file_name, silence=True)
+
         self.topology = Topology(name=Path(self.plan_pro_file_name).stem)
+        self.topology.created_at = self._get_created_at()
+        self.topology.created_with = self._get_created_with()
+
+    def _get_created_at(self) -> datetime:
+        """Gets the date object, when the PlanPro was created
+
+        :return: The date object
+        """
+        return self.root_object.PlanPro_Schnittstelle_Allg.Erzeugung_Zeitstempel.Wert
```

Comment:
Assuming `.Wert` returns a value converted to `datetime`.

Reply:
Yes, that works directly.